src/views/virtualmachines/details/tabs/configuration/network/components/list/NetworkInterfaceList.tsx

```diff
@@ -63,7 +57,7 @@ const NetworkInterfaceList: FC<NetworkInterfaceTableProps> = ({ vm }) => {
         loaded={!isEmpty(vm)}
         loadError={false}
         Row={NetworkInterfaceRow}
-        rowData={{ vm }}
+        rowData={{ vm, vmi }}
```

Comment:
isnt it easier just to send isPending instead of vmi ? and do getVMINetworks(vmi)?.some((ntwork) => ntwork?.name !== network.name); here?

Reply:
Do you mean pass the function?

Reply:
rowData={{ vm, isPending }}

Reply:
Done. Good suggestion. =)